src/WebJobs.Script/Binding/ServiceBusScriptBindingProvider.cs

```diff
@@ -140,32 +138,21 @@ public override Collection<Attribute> GetAttributes()
 
                 if (Context.IsTrigger)
                 {
-                    attributes.Add(new EventHubTriggerAttribute(eventHubName));
-
-                    string eventProcessorHostName = Guid.NewGuid().ToString();
-                    string storageConnectionString = _storageConnectionString;
-
+                    var attribute = new EventHubTriggerAttribute(eventHubName);
                     string consumerGroup = Context.GetMetadataValue<string>("consumerGroup");
-                    if (consumerGroup == null)
+                    if (consumerGroup != null)
```

Comment:
to catch what? both old and new behavior here are correct.  We don't have any end-2-end tests on consumer groups. (CG have to be created in the portal; so it's one more external dependency).